src/System.Linq/src/System/Linq/Repeat.cs

```diff
@@ -79,10 +79,7 @@ public TResult[] ToArray()
                 TResult[] array = new TResult[_count];
                 if (_current != null)
```

Comment:
Yeah, it'll box. And now I think about it, I wrote that skip on null, and that's precisely why I didn't test for default generally. I'd certainly love to learn of a non-boxing mechanism.